var/spack/repos/builtin/packages/qt-declarative/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class QtDeclarative(CMakePackage):
+    """Qt Declarative (Quick 2)."""
+
+    homepage = "https://www.qt.io"
+    url      = "https://github.com/qt/qtdeclarative/archive/refs/tags/v6.2.3.tar.gz"
+    list_url = "https://github.com/qt/qtdeclarative/tags"
+
+    maintainers = ['wdconinc', 'sethrj']
+
+    version('6.2.3', sha256='eda82abfe685a6ab5664e4268954622ccd05cc9ec8fb16eaa453c54900591baf')
+
+    generator = 'Ninja'
+
+    # Changing default to Release for typical use in HPC contexts
+    variant('build_type',
+            default='Release',
+            values=("Release", "Debug", "RelWithDebInfo", "MinSizeRel"),
+            description='CMake build type')
+
+    depends_on('cmake@3.16:', type='build')
+    depends_on('ninja', type='build')
+    depends_on("pkgconfig", type='build')
+    depends_on("python", when='@5.7.0:', type='build')
+
+    versions = ['6.2.3']
+    for v in versions:
```

Comment:
Best to use 'private' variables for class-level temporaries because they become part of the package. You don't want someone to add a `def versions(self)` to the package and conflict with it.

```suggestion
    _versions = ['6.2.3']
    for v in _versions:
```

Reply:
I'll propagate this change to the other places it is used too.